lib/hanami/configuration.rb

```diff
@@ -33,7 +36,12 @@ class Configuration
     attr_reader :environments
     private :environments
 
-    def initialize(env:)
+    attr_reader :application_name
+
+    def initialize(application_name:, env:)
```

Comment:
Might be better to have this `application_name` param actually called `application_class_name` or something, since that's what it is at this point? i.e. it's not yet transformed into the actual concept of the "application name" that we otherwise expect throughout the system.

Reply:
@timriley Nope, this is already the `application_name`.

For a given app defined by `MyApp::Application`, we receive as argument here the underscored version of the namespace: `my_app`.